### PR TITLE
Chinese annotation

### PR DESCRIPTION
**Pull request by VisualSJ**

### CODE REVIEW

**Comment by VisualSJ** [comment]
@natural-law Please review this pr

File: cocos/base/CCValue.h [hunk 1]
```diff
@@ -49,140 +49,166 @@ CC_DLL extern const ValueMap ValueMapNull;
 CC_DLL extern const ValueMapIntKey ValueMapIntKeyNull;
 
 /*
- * This class is provide as a wrapper of basic types, such as int and bool.
+ * @~english This class is provide as a wrapper of basic types, such as int and bool.
+ * @~chinese 这个类提供wrapper的基本类型,如int和bool。
```
Comment (natural-law) [inline cocos/base/CCValue.h]: 应该是：基本类型的 wrapper

File: cocos/base/CCValue.h [hunk 2]
```diff
@@ -49,140 +49,166 @@ CC_DLL extern const ValueMap ValueMapNull;
...
+    /** @~english Gets as a const ValueMapIntKey reference. Will convert to ValueMapIntKey if possible, or will trigger assert error.  @~chinese 获取一个常量ValueMapIntKey。如果可能将转换为ValueMapIntKey,否则引发断言错误。*/
     const ValueMapIntKey& asIntKeyMap() const;
 
     /**
-     * Checks if the Value is null.
-     * @return True if the Value is null, false if not.
+     * @~english Checks if the Value is null.
+     * @~chinese  检查值是否为null。
+     * @return @~english True if the Value is null, false if not.
+     * @~chinese 真如果该值为null则为true,否则为false。
```
Comment (natural-law) [inline cocos/base/CCValue.h]: 多了一个“真”

**Comment by VisualSJ** [comment]
@natural-law Please review this pr , thanks

**Comment by natural-law** [comment]
@andyque 
It's OK for merge.

**Pull request merged**